### DISCUSSION TITLE
Feedback for the "Synesthesia" mod

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#24110] [Ideas]
Hello, here is some feedback for the "Synesthesia" mod:

1. https://github.com/ppy/osu/pull/23520#issuecomment-1550061287 – I agree with this comment, this being a gameplay mod or its own  setting should be consistent between different rulesets. Personally, I think it should be a mod as it already has a nice name and description in the mod list.

2. Speaking of the name of this mod, it could be renamed from "Synesthesia" to "Chromesthesia". The current name is fine, but this would possibly be a more descriptive name for the mod. See https://en.wikipedia.org/wiki/Chromesthesia

3. In https://osu.ppy.sh/wiki/en/Ranking_criteria/osu%21

> Avoid using combo colours with ~220 luminosity or higher during kiai times. They create bright pulses which can be unpleasant to the eyes.

This mod changes the colors of a good portion of the hit objects to white, which can apparently be uncomfortable for some players. 

Possible solutions:

* See https://github.com/ppy/osu/discussions/24036 – if this is implemented, then players can change the white color to a darker color.
* Players could use "combo color normalisation" to make the colors less bright, although it looks like this is not currently implemented for this mod, as also said in this comment: https://github.com/ppy/osu/pull/23520#issuecomment-1569701744.
* Of course, players can always opt out of using this mod if they feel that the pulsing is unpleasant.

**Comment by hatchywyd** [comment]
might be an unpopular opinion, but i think synesthesia is low-key cheating.

there are so many sections that i can hit much easier without having to put in the normal amount of effort needed to read certain patterns.

i thought about suggesting nerfs, but then i realized it should just be removed entirely. 

reading patterns should be determined by player skill and experience; not a mod that does all the work for you.

**Reply by verarr** [reply]
You could argue that this is a new way to read patterns; it doesn't click nor aim for you. Sometimes I too get the feeling that using SY is cheating, but most of the time the difficulty of a beatmap doesn't come from stupid rhythm patterns, with which SY _does_ help.

If I were to add a multiplier to SY, it would be 0.97 so that it roughly cancels out with half of HD (because HD makes two things harder: aim and timing, SY helps with timing, so half of HD is canceled out; this probably sounds very stupid though)

By the looks of it, though, SY definitely isn't getting removed now. Maybe, it will stay unranked.

**Reply by huyenden** [reply]
I played with the Synesthesia mod for half a year since Synesthesia was in version 2023.621.0. And to be honest, they are not much different from normal. For me, they just make the stream look better, so it can't be considered cheating or buffing the accuracy

**Reply by peppy** [reply]
It should definitely have a debuff multiplier as it alters readability.

**Reply by huyenden** [reply]
Anyway, I still like SY better, no problem with Synesthesia.

**Comment by peppy** [comment]
The name sucks. Needs to be renamed to something not so awkward.

**Comment by doginuu** [comment]
It'd be nice if the colors were customizable via skin.ini, I'm not really sure how the ini works in lazer but I do know my combo colors from my skin are transferred. Synesthesia could just match the "Combo1" to snap 1/1, "Combo3" to snap 1/3 and so on... Maybe give it's own category like "SYCombo1" instead whatever works. I think the mod is a very good feature and I hope it becomes more customizable and realized but right now the colors to me are just confusing since I don't know timeline ticks by heart and the colors are a bit of an eyesore with how dense they are (at least on my skin that feels like it's better with soft colors).

One of my least favorite things about reading in general is trying to distinguish close rhythms like ticks: white, 2nd blue, next red, white vs triplets, I tend to find it more map knowledge rather than being able to discern a few ms difference between the rates the 2nd note's approach circle comes in (for that pattern in particular it's the last note that gives away if it's the first rhythm). I like the idea of making snaps of 1, 2, 4, etc. similar or the same colors then contrasting that previous scheme is 3, 6, 9, etc. following another scheme, same for the rest of the odd snaps like 5, 7, etc.

Another potential idea for the mod would be a toggleable option to add sliderbody tint close to the sliderhead that indicates a slider velocity increase or decrease from the previous slider (blue tint for decrease, red tint for increase, stronger color intensity depending on how drastic the SV change is). I really like the idea of having more sightread potential rather than replaying or being at the mercy of a mapper to not make anything too unintuitive.

I think this mod has a lot of potential, it's a little disingenuous to say the mod only removes reading skill because it's a huge quality of life if you misread/didn't understand a rhythm and immediately opened the editor to see what you just looked at. In that case it didn't remove reading skill, it removed the skillcheck sure but also the time of you opening the editor to memorize a single pattern and play the map again. Even at x0.8 I'd love to always use this mod and just take it off if I care to try on leaderboards, I really enjoy stuff like this from dance games.

**Reply by huyenden** [reply]
Agree that if you can change the color from skin.ini it will be more convenient. Like how slidermissend.png is added
